### DISCUSSION TITLE
Move or duplicate pagination to the top slot

### DISCUSSION THREAD

**Discussion by marinovdf** [quasarframework/quasar#11617] [General - Components / Directives / etc]
Hi!
I use `QTable` component with server side pagination. How can I move or duplicate pagination to the `top-right` slot without rewriting all pagination.
Perfectly I want to use `pagination` slot above and below table.

In Quasar 1.* `TablePagination` was a separate component. But in versions 2.* pagination fixed in `QTable` component.

I use: Vue 3, Quasar 2.3.4

**Comment by metalsadman** [comment]
i tried it in the past but on qv1, https://codepen.io/metalsadman/pen/WNRapWe, `extends` afaik doesn't work as it used to in vue 3.

**Reply by marinovdf** [reply]
Thanx for your answer. I will try to make something like that for qv2 and vue3.

**Reply by paulocoutinhox** [reply]
Any solution for this?

**Comment by ghost** [comment]
Hi there, I created a custom TablePagination.vue component (see [the following Gist](https://gist.github.com/EfChouTR/e05198263f1d291829d6e05006a65c4f)) that imports a lot of the quasar src code.

The only caveat is you will be responsible for adding it to the #top slot... I dont think it would be too difficult to add a clause to render the pagination element at top/bottom (default)/both in their QTable code depending on the passed props.  To make it simple, I would add a new prop called "paginationPlacement" with the default as "bottom" and the options as top | bottom | both
